runtime/parachains/src/session_info.rs

```diff
@@ -37,6 +40,10 @@ pub mod migration;
 #[cfg(test)]
 mod tests;
 
+// The order of parameters should be deterministic, that is, one should not reorder them when
+// changing the array contents to avoid creating excessive pressure to PVF execution subsys.
+const EXECUTOR_PARAMS: [ExecutorParam; 0] = [];
```

Comment:
BTW, why isn't this part of the parachains configuration? Why is this some constant here?

Reply:
I look at it this way: it's part of the `session_info` pallet (and was supposed to be a part of `SessionInfo` itself at some point), and the `session_info` pallet is a part of the parachain host.

But you're probably right, `configuration` may be a better place for it. But it's a temporary solution anyway, as in the end, we want to be able to control executor parameters through governance, not only through runtime upgrades. So IIUC, we'll need to move it to `configuration` anyway and then provide some methods to schedule those config upgrades (I'm just not familiar with that part yet).

Reply:
Raised #6805 on that.

Reply:
Okay ty!